devtools/gradle/gradle-application-plugin/src/test/java/io/quarkus/gradle/tasks/CachingTest.java

```diff
@@ -97,8 +96,8 @@ void envChangeInvalidatesBuild() throws Exception {
         assertBuildResult("change FOO_ENV_VAR rebuild", gradleBuild(arguments, env), ALL_UP_TO_DATE);
 
         // Change an unrelated environment variable, all up-to-date
-        env = Map.of("SOME_UNRELATED", "meep");
-        assertBuildResult("SOME_UNRELATED", gradleBuild(arguments, env), FROM_CACHE);
+        env = Map.of("FOO_ENV_VAR", "some-other-value", "SOME_UNRELATED", "meep");
```

Comment:
if we don't add the previous variable `FOO_ENV_VAR`, there will be a configuration cache miss because the variable has changed, what we are testing here is that `SOME_UNRELATED` is not affecting the build.